examples/tp_table.rs

```diff
@@ -0,0 +1,208 @@
+            line.push(format!("{:.2}±{:.2}", (&tps).mean(), tps.std_dev()));
+        }
+        printed.push(line);
+    }
+    printed
+}
+
+fn analyze_adj(data: &[Vec<(usize, Vec<f64>)>], rc_vec: &[usize]) -> Vec<Vec<String>> {
+    let mut printed = Vec::with_capacity(data.len());
+    for (line_data, rc) in data.iter().zip(rc_vec) {
+        let mut line = Vec::with_capacity(line_data.len() + 1);
+        line.push(rc.to_string());
+        let n_frames_duration_0 = OnceCell::new();
+        for (n_frames, durations) in &line_data[1..] {
+            let (n_frames_0, duration_0) = n_frames_duration_0.get_or_init(|| {
+                let (n_frames_0, durations_0) = &line_data[0];
+                let duration_0 = durations_0.mean();
+                (*n_frames_0, duration_0)
+            });
+            let tps = durations
+                .iter()
+                .map(|d| (n_frames - n_frames_0).to_f64().unwrap() / (d - duration_0))
+                .collect::<Vec<_>>();
+            line.push(format!("{:.2}±{:.2}", (&tps).mean(), tps.std_dev()));
+        }
+        printed.push(line);
+    }
+    printed
+}
+
+fn rc_env() -> Result<Vec<usize>> {
+    std::env::var("LURK_RC")
+        .map_err(|e| anyhow!("LURK_RC env var isn't set: {e}"))
+        .and_then(|rc| {
+            let vec: Result<Vec<usize>> = rc
+                .split(',')
+                .map(|rc| {
+                    rc.parse::<usize>()
+                        .map_err(|e| anyhow!("Failed to parse RC: {e}"))
+                })
+                .collect();
+            vec
+        })
+}
+
+fn n_folds_env() -> Result<usize> {
+    std::env::var("LURK_N_FOLDS")
+        .map_err(|e| anyhow!("LURK_N_FOLDS env var isn't set: {e}"))
+        .and_then(|n_folds| {
+            n_folds
+                .parse::<usize>()
+                .map_err(|e| anyhow!("Failed to parse N_FOLDS: {e}"))
+        })
+}
+
+fn main() {
+    let rc_vec = rc_env().unwrap_or_else(|_| vec![100]);
+    let max_n_folds = n_folds_env().unwrap_or(3);
+    let n_samples = 10;
+
+    let max_rc = rc_vec.iter().max().unwrap();
+
+    let limit = n_iters(max_n_folds, *max_rc);
+
+    let store = Store::<Fr>::default();
+    let program = store.read_with_default_state(PROGRAM).unwrap();
+
+    let (frames, _) = evaluate::<Fr, Coproc<Fr>>(None, program, &store, limit).unwrap();
+
+    let lang = Lang::<Fr, Coproc<Fr>>::new();
+    let lang_arc = Arc::new(lang.clone());
+
+    let mut data = Vec::with_capacity(rc_vec.len());
+
+    let n_folds_vec = (0..=max_n_folds).collect::<Vec<_>>();
+
+    for rc in rc_vec.clone() {
+        let prover: NovaProver<'_, _, _, MultiFrame<'_, _, Coproc<Fr>>> =
+            NovaProver::new(rc, lang.clone());
+        println!("Getting public params for rc={rc}");
+        // TODO: use cache once it's fixed
+        let pp: PublicParams<_, MultiFrame<'_, _, Coproc<Fr>>> =
+            public_params(rc, lang_arc.clone());
+        let n_folds_data = n_folds_vec
```

Comment:
Can't you just use `(0..=max_n_folds)` here literally? Capturing in a vector above seems superfluous.